components/scroll-wrapper/scroll-wrapper.js

```diff
@@ -0,0 +1,222 @@
+import '../colors/colors.js';
+import '../icons/icon.js';
+import { css, html, LitElement } from 'lit-element/lit-element.js';
+import { FocusVisiblePolyfillMixin } from '@brightspace-ui/core/mixins/focus-visible-polyfill-mixin.js';
+import ResizeObserver from 'resize-observer-polyfill/dist/ResizeObserver.es.js';
+import { RtlMixin } from '../../mixins/rtl-mixin.js';
+
+const RTL_MULTIPLIER = navigator.userAgent.indexOf('Edge/') > 0 ? 1 : -1; /* legacy-Edge doesn't reverse scrolling in RTL */
```

Comment:
Previously this logic was... quite complex. Legacy-Edge now appears to be the only browser that has weird RTL scrolling behaviour, so I've limited the hackiness to just check for it.